### PR TITLE
Unify logging in xup/*

### PR DESCRIPTION
**Pull request by aquesnel**
Follow-on pull request to #1633 which migrates all logging in the xup directory to use the LOG() and LOG_DEVEL() macros.

Changes:

* Migrate logging to the LOG() and LOG_DEVEL() macros (using text transform script)
* error logs use the LOG macro
* update code formatting with astyle for all files in xup/*

### CODE REVIEW

File: xup/xup.c [hunk 1]
```diff
@@ -1446,14 +1437,14 @@ lib_mod_process_message(struct mod *mod, struct stream *s)
     int type;
     char *phold;
 
-    LLOGLN(10, ("lib_mod_process_message:"));
+    LOG_DEVEL(LOG_LEVEL_DEBUG, "lib_mod_process_message:");
```
Comment (matt335672) [inline xup/xup.c:1440]: LOG_LEVEL_TRACE?

**Review by matt335672 — COMMENTED** [review]
Minor comment. I think we can merge this one when we've discussed.

Thanks again.

File: xup/xup.c (hunk 1 above)
Comment (aquesnel) [reply xup/xup.c]: fixed

**Comment by aquesnel** [comment]
@matt335672, I've implemented the feedback you suggested and review the log level for the rest of the log messages.

I'm still investigating the freebsd build failure when building the latest commit.

**Comment by matt335672** [comment]
Thanks for that.

The FreeBSD build failure's a bit on the odd side. I've just tried re-running your job here:-

https://cirrus-ci.com/task/6336619461476352

It's still failing as of 2020-02-10 09:17 UTC

I need to find out more about FreeBSD anyway so I'll try knocking up a VM here and see what happens (it may take me a while). If it compiles fine locally we can at least carry on with other work for now. I agree that it seems very unlikely indeed to have anything to do with your PR.

**Comment by matt335672** [comment]
Well I can confirm that your branch cloned with the following command builds fine on FreeBSD 12.2:-

```
git clone -b unify_logging_xup --depth 1 https://github.com/aquesnel/xrdp.git build
```

@metalefty - given the circumstances are you happy for me to merge this? Or do you want to wait until you've investigated further?

**Comment by matt335672** [comment]
@aquesnel - please rebase to latest devel and squash. That should fix it, and then I can merge this in.

Thanks.

**Comment by aquesnel** [comment]
@matt335672 Thanks for fixing the issue with the FreeBSD build. 
I've rebased and squashed the changes.

**Comment by matt335672** [comment]
Great - thanks again!

**Pull request merged**